crates/bitwarden-core/src/client/encryption_settings.rs

```diff
@@ -96,8 +96,13 @@ impl EncryptionSettings {
     ) -> Result<(), EncryptionSettingsError> {
         let mut ctx = store.context_mut();
 
+        // FIXME: [PM-11690] - Early abort to handle private key being corrupt
+        if org_enc_keys.is_empty() {
+            return Ok(());
+        }
```

Comment:
This is duplicated?

Reply:
Oops, looks like I accidentally moved it during the rebase and then didn't notice it was already down there, fixed in 9b50890